docs/setup.md

```diff
@@ -34,13 +34,13 @@ Get your OpenAI API key from: [https://platform.openai.com/account/api-keys](htt
 ### Set up with Docker
 
 1. Make sure you have Docker installed, see [requirements](#requirements)
-2. Pull the latest image from [Docker Hub]
+2. Create a project directory for Auto-GPT
 
         :::shell
-        docker pull significantgravitas/auto-gpt
+        mkdir Auto-GPT
+        cd Auto-GPT
 
-3. Create a folder for Auto-GPT
-4. In the folder, create a file called `docker-compose.yml` with the following contents:
+3. In the project directory, create a file called `docker-compose.yml` with the following contents:
 
         :::yaml
```

Comment:
":::yaml" this should not be added to the file contents. When opening setup.md from folder this seems part of the code